docs/changelog/6_x.rst

```diff
@@ -0,0 +1,219 @@
+process:
+
+1. EdgeDB v6.0 supports PostgreSQL 14 or above. Verify your PostgreSQL version
+   before upgrading EdgeDB. If you're using Postgres 13 or below, upgrade
+   Postgres first.
+
+2. Spin up an empty 6.0 instance. You can use one of our :ref:`deployment
+   guides <ref_guide_deployment>`.
+
+   For Debian/Ubuntu, when adding the EdgeDB package repository, use this
+   command:
+
+   .. code-block:: bash
+
+       $ echo deb [signed-by=/usr/local/share/keyrings/edgedb-keyring.gpg] \
+           https://packages.edgedb.com/apt \
+           $(grep "VERSION_CODENAME=" /etc/os-release | cut -d= -f2) main \
+           | sudo tee /etc/apt/sources.list.d/edgedb.list
+       $ sudo apt-get update && sudo apt-get install edgedb-6
+
+   For CentOS/RHEL, use this installation command:
+
+   .. code-block:: bash
+
+       $ sudo yum install edgedb-6
+
+   In any required ``systemctl`` commands, replace ``edgedb-server-6`` with
+   ``edgedb-server-6``.
+
+   For Docker setups, use the `6.0` tag.
+
+3. Take your application offline, then dump your v5.x database with the CLI:
+
+   .. code-block:: bash
+
+       $ edgedb dump --dsn <old dsn> --all --format dir my_database.dump/
+
+   This will dump the schema and contents of your current database to a
+   directory on your local disk called ``my_database.dump``. The directory name
+   isn't important.
+
+4. Restore the empty v6.x instance from the dump:
+
+   .. code-block:: bash
+
+       $ edgedb restore --all my_database.dump/ --dsn <new dsn>
+
+   Once the restore is complete, update your application to connect to the new
+   instance.
+
+   This process will involve some downtime, specifically during steps 2 and 3.
+
+
+New features
+============
+
+SQL write support
+-----------------
+
+You can now use SQL DML (``insert``, ``update``, ``delete``) when connecting to
+your EdgeDB instance via the PostgreSQL protocol. Our aim is to support most
+typical use cases from tools like SQL ORMs and SQL clients.
+
+This allows more developers to use EdgeDB, leveraging our advanced data model,
+tooling, and high-performance connection management. Teams can migrate their
+existing SQL codebases to EdgeDB without rewriting their queries. Once adopted,
+you can gradually take advantage of EdgeQL's powerful query capabilities.
+
+Existing EdgeDB users who already use EdgeQL can benefit too. While some SQL
+features like window functions, recursive queries, and explicit locking are not
+yet supported, you can use these features in SQL today. We will continue to add
+support for more features in the future.
+
+In-place upgrade
+----------------
+
+We aim for this version to be the last requiring a full dump and restore
+process for major version upgrades. We understand that dump-and-restore is
+disruptive, so enabling in-place upgrades will make it easier for teams to
+upgrade more frequently.
+
+Query performance observability
+-------------------------------
+
+We now store statistics about query performance.
```

Comment:
Can we add a screenshot here? Also would be great to expand a bit, like that client APIs now have `withQueryTag` method to apply custom perf tags to your queries in your code

Reply:
The UI for this isn't available in the beta yet. Do you think it still makes sense to show a screenshot now?